src/main/resources/db/changelog/changes/2-ays-dml.xml

```diff
@@ -106,12 +106,30 @@
             <column name="LAST_NAME" value="Sistemi"/>
             <column name="COUNTRY_CODE" value="90"/>
             <column name="LINE_NUMBER" value="1234567890"/>
+            <column name="ROLE" value="ADMIN"/>
+            <column name="STATUS" value="ACTIVE"/>
+            <column name="CREATED_USER" value="AYS"/>
+            <column name="CREATED_AT" valueDate="now()"/>
+        </insert>
+
+        <insert tableName="AYS_ADMIN_USER">
+            <column name="ID" value="9262f0fc-93db-4f7e-81c6-aaad85c2b299"/>
+            <column name="ORGANIZATION_ID" value="null"/>
+            <column name="USERNAME" value="ays-super-admin"/>
+            <column name="PASSWORD" value="$2a$12$tvvL2n1XLYkYX3X1PRI5Bu35zXXvM4MEgE6jU3.38bqVil3aiGmQS"/>
```

Comment:
If the passwords are the same as the user mentioned above, it would be much more flexible and user-friendly. Since default users are already defined, it would make it easier to use.

Reply:
I just revised the `password ` field. Can we look through it again?